api/build.gradle

```diff
@@ -388,26 +388,26 @@ dependencies {
     compile "com.github.java-json-tools:json-patch:1.13"
     compile "com.github.rholder:guava-retrying:2.0.0"
     compile "org.javers:javers-core:5.10.1"
-    compile "com.google.api-client:google-api-client-appengine:1.30.0"
-    compile "com.google.apis:google-api-services-admin-directory:directory_v1-rev20190806-1.30.3"
-    compile "com.google.apis:google-api-services-cloudbilling:v1-rev48-1.25.0"
-    compile "com.google.apis:google-api-services-cloudresourcemanager:v1-rev549-1.25.0"
-    compile "com.google.apis:google-api-services-oauth2:v2-rev139-1.23.0"
+    compile "com.google.api-client:google-api-client-appengine:1.32.2"
+    compile "com.google.apis:google-api-services-admin-directory:directory_v1-rev20211109-1.32.1"
+    compile "com.google.apis:google-api-services-cloudbilling:v1-rev20211106-1.32.1"
+    compile "com.google.apis:google-api-services-cloudresourcemanager:v1-rev572-1.25.0"
```

Comment:
`cloudresourcemanager` and `oauth2` have upgrades beyond these versions but they're breaking changes